GPXLab/gpxlab.cpp

```diff
@@ -102,7 +102,7 @@ GPXLab::GPXLab(const QString &fileName, QWidget *parent) :
     connect(zoomSlider, SIGNAL(valueChanged(int)), this, SLOT(zoom_valueChanged(int)));
 
     // exit
-    ui->actionExit->setShortcut(QKeySequence::Quit);
+    ui->actionExit->setShortcut(QKeySequence(tr("Ctrl+Q")));
```

Comment:
What is wrong with QKeySequence::Quit here? That is standard shortcut (depends on platform), translation is actuallly not needed.

Reply:
As per Qt documentation  (https://doc.qt.io/qt-5/qkeysequence.html) under windows is not assigned, so it is not working.

Reply:
I guess that's because there is `Alt+F4` shortcut on Windows, that works for all windows. So `Ctrl+Q` would just duplicate it.

Reply:
Probably, even if I don't think `Alt+F4` is a real substitute. 
But at least `Ctrl+Q` under Windows add consistency with the other platforrms.

Reply:
Here my suggestion:
```
  #ifdef Q_OS_WINDOWS
    ui->actionExit->setShortcut(QKeySequence("Ctrl+Q"));
  #else
    ui->actionExit->setShortcut(QKeySequence::Quit);
  #endif
```
If no opposition I will implement it that way later today or tomorrow.